libs/shared/src/consts/providers/channels/email.ts

```diff
@@ -21,6 +21,14 @@ import { EmailProviderIdEnum } from '../provider.enum';
 import { ChannelTypeEnum } from '../../../types';
 
 export const emailProviders: IProviderConfig[] = [
+  {
+    id: EmailProviderIdEnum.Novu,
+    displayName: 'Novu Email',
+    channel: ChannelTypeEnum.EMAIL,
+    credentials: [],
+    docReference: 'https://docs.novu.co/channels/email/novu-email',
+    logoFileName: { light: 'novu.png', dark: 'novu.png' },
+  },
```

Comment:
add Novu Email provider to the email providers array...
this fixes the issue for icon for Novu Email/SMS integration when multi providers ff is on